migrate.py

```diff
@@ -186,13 +186,16 @@ def main(options):
             # former had a bunch more comments to be processed.
             # https://github.com/jeffwidman/bitbucket-issue-migration/issues/45
             status_url = push_respo.json()['url']
-            gh_issue_url = verify_github_issue_import_finished(
-                status_url, options.gh_auth, headers).json()['issue_url']
+            resp = verify_github_issue_import_finished(
+                status_url, options.gh_auth, headers)
+
             # verify GH & BB issue IDs match
             # if this fails, convert_links() will have incorrect output
             # this will fail if the GH repository has pre-existing issues
-            gh_issue_id = int(gh_issue_url.split('/')[-1])
-            assert gh_issue_id == issue['local_id']
+            if resp:
```

Comment:
Could use a comment right here explaining why we don't assume a response... something as simple as "Github occasionally inexplicably returns 404 even when successfully importing an issue. For details see #77"